### DISCUSSION TITLE
"Unable to verify validity of certificate" Error when testing the email Configuration

### DISCUSSION THREAD

**Discussion by stefans007** [globaleaks/globaleaks-whistleblowing-software#3582] [Q&A] [answered]
Dear Globaleaks Devs,

We have set up our site with a self signed certificate.
When configuring and testing the email setup, the log shows an error when trying to send a test mail:
"Unable to verify validity of certificate: <info of our root certificate>"
Email hence does not work.
How does Globaleaks verify this validity here?
Do we need a root certificate from an official certification authority ?

Thanks for your help.

**Comment by evilaliv3** [comment, marked as answer]
Hello!
GlobaLeaks uses the system trust store to validate the certificate chain.

To use a self signed certificate you should add your certificate to the trust store: https://ubuntu.com/server/docs/security-trust-store

**Comment by stefans007** [comment]
Thanks a lot.
